### DISCUSSION TITLE
How run tests in SFC Playground?

### DISCUSSION THREAD

**Discussion by pasikonik** [webfansplz/vuejs-challenges#1453] [General]
Hi

I'm wondering how I can run tests in SFC Playground?
Is it even possible?

**Comment by wangyinyuan** [comment]
I'm experiencing the same confusion, there doesn't seem to be an associated button in the Vue SFC Playground that can run a test, nor is there a command line.But I do found out a solution for lazy person (like me).

The author of this repo has put the project documentation on a website, where **the Chinese version** gives a new link for each challenge, jumping to the "stackblitz" website to run the tests automatically, the link to this documentation is: "https://cn-vuejs-challenges.netlify.app/questions/1-hello-word/readme.zh-cn.html"

![image](https://github.com/webfansplz/vuejs-challenges/assets/110143758/9f32e957-659e-4c0a-b090-d49aa65699c4)

so you can modify App.vue in this website for auto test.

**Hope will help**
